Chap_Terms.tex

```diff
@@ -20,27 +20,34 @@ \chapter{PMIx Terms and Conventions}
 
-\item \declareterm{application} refers to a single executable (binary, script, etc.) member of a \emph{job}.
+\item \declareterm{namespace} refers to a character string value assigned by the \ac{RM} to a \textit{job}.  The requester of the \textit{job} (e.g., \code{mpiexec} or a \ac{PMIx} client) may request a specific \emph{namespace} value.  All \textit{applications} executed as part of that \textit{job} share the same \emph{namespace}. The \emph{namespace} assigned to each \emph{job} must be unique within the scope of the governing \ac{RM} and often is implemented as a string representation of the numerical emph{Job ID}. The \emph{namespace} and \emph{job} terms will be used interchangeably throughout the document.
+
+\item \declareterm{application} represents a set of identical, but not necessarily unique,
+execution contexts within a \emph{job}.
 
 \item \declareterm{process} refers to an operating system process, also commonly referred to as a \emph{heavyweight} process. A process is often comprised of multiple \emph{lightweight threads}, commonly known as simply \declaretermAlt{threads}{thread}.
 
 \item \declaretermAlt{client}{clients} refers to a process that was registered with the \ac{PMIx} server prior to being started, and connects to that \ac{PMIx} server via \refapi{PMIx_Init} using its assigned namespace and rank with the information required to connect to that server being provided to the process at time of start of execution.
 
 \item \declaretermAlt{clone}{clones} refers to a process that was directly started by a \ac{PMIx} client (e.g., using \emph{fork/exec}) and calls \refapi{PMIx_Init}, thus connecting to its local \ac{PMIx} server using the same namespace and rank as its parent process.
 
-\item \declaretermAlt{slot}{slots} refers to an allocated entry for a process. \acp{WLM} frequently allocate entire nodes to a \emph{session}, but can also be configured to define the maximum number of processes that can simultaneously be executed on each node. This often corresponds to the number of hardware \acp{PU} (typically cores, but can also be defined as hardware threads) on the node. However, the correlation between hardware \acp{PU} and slot allocations strictly depends upon system configuration.
+\item \declaretermAlt{slot}{slots} refers to an allocated set of resources assigned to a process.  \acp{WLM} frequently allocate entire nodes to a \emph{session}, but can also be configured to define the maximum number of processes that can simultaneously be executed on each node. This often corresponds to the number of hardware \acp{PU} (typically cores, but can also be defined as hardware threads) on the node. However, the correlation between hardware \acp{PU} and slot allocations strictly depends upon system configuration.
```

Comment:
Per ASC Meeting Oct. 1, 2020:
 * Colloquially, a `slot` equals one process "slot" on the node
   * `slot` is the number of processes that you are run on a node.
   * It does not mean core or hwthread (or any specific set of resources). A slot is independent of the resource consumption of the process being lanched there.
   * If oversubscription (and it's allowed by the system), then you are assigning more than one process per slot.
 * Maybe rephrase to "A slot refers to used to define the available space for a single process on a node" and "so the number of slots on a node is the number of processes that can run on a node"? (or "within the specified realm" instead of "node")
 * Is it useful to define a term (e.g., "resource set") that is tied to the resources assigned to a process?
 * The working group will need to discuss this topic further before next reading.